cypress/support/commands.ts

```diff
@@ -42,8 +42,9 @@ Cypress.Commands.add(
 );
 
 Cypress.Commands.add('waitForJs', function () {
-  cy.window().its('Cypress').should('be.an', 'object');
-  cy.window().should('have.property', 'Ready', true);
+  // cy.window().its('Cypress').should('be.an', 'object');
+  // cy.window().should('have.property', 'Ready', true);
+  cy.wait(1900);
```

Comment:
Is there no other way to detect that the page is ready?

Reply:
The commented code is the recommended way to handle it, the problem is that it is extremely inconsistent in no-head mode. This may have been resolved by newer versions :eyes:

Reply:
Hmm, I see. Might be worth testing, 1900ms is a loong wait